app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    def initialize(keyword_id:)
+      @keyword_id = keyword_id
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def call!(search_service: nil)
```

Comment:
[As commented on Slack](https://nimble-co.slack.com/archives/C05HAMT941Z/p1690897046668539), `search_service` should be added as a param in the constructor. Also, we can likely default to `Google::SearchService` if no param is provided:

```ruby
def initialize(keyword_id:, search_service: Google::SearchService)
```

Reply:
Fixed in 6fbb5e0caa34d9cb91ab35a3f314d40cac37eb95